BepInEx.Core/Contract/PluginInfo.cs

```diff
@@ -77,7 +90,6 @@ void ICacheable.Save(BinaryWriter bw)
     void ICacheable.Load(BinaryReader br)
     {
         TypeName = br.ReadString();
-        Location = br.ReadString();
```

Comment:
Sidenote: this (I think?) will crash on older caches and result in skipping the cache via the try-catch, but should we make this versioned so we can skip on cache format updates more nicely?

Reply:
I have added versioning at the CachedAssembly level so we avoid throwing in the case the cache changes